ddtrace/opentracer/tracer.py

```diff
@@ -78,7 +78,6 @@ def __init__(self, service_name=None, config=None, scope_manager=None, dd_tracer
                                   """)
 
         self._scope_manager = scope_manager or ThreadLocalScopeManager()
-
```

Comment:
did you run black or is this unrelated?

Reply:
unrelated, I think was just an artifact of moving code around